app/api/currentuser/accept_group_invitation.go

```diff
@@ -24,7 +24,9 @@ import (
 //   required: true
 // responses:
 //   "201":
-//     "$ref": "#/responses/updatedOrNotChangedResponse"
+//     "$ref": "#/responses/updatedResponse"
+//   "205":
```

Comment:
If you do not change tests and do not change the service to return 205, the behaviour is not changed, isn't it?
BTW, I don't get why 205 meaning "Reset Content" (you even use the ` http.StatusResetContent` constant for it) is used for "unchanged".

Reply:
I didn't change the service. I changed only its swagger specs. The service always returned 205 as 'not changed'. Unfortunately, there is no such HTTP status "Not Changed" so many people use 205.

Reply:
Would you have a reference for that?

Reply:
The only link I can find now is this one: https://stackoverflow.com/a/14224164
I thought this way: 1) 205 is an acceptable status code, 2) Most of the time, the "not changed" response is caused by stale data in a browser so we should refresh the data. Although refresh != reset, it is still better than just "OK"

Reply:
The issue with 205 is `Since the 205 status code implies that no additional content will be
   provided, a server MUST NOT generate a payload in a 205 response.`

Reply:
Should we return 200 for "not changed"?

Reply:
fixed (will be 200 for PUT/DELETE and 201 for POST)